server/src/grpc/meta_event_service/error.rs

```diff
@@ -22,13 +22,17 @@ pub enum Error {
         msg: String,
         source: GenericError,
     },
+
+    #[snafu(display("Open shard error, code:{:?}, message:{}", code, msg))]
+    OpenShardErr { code: StatusCode, msg: String },
```

Comment:
Reuse ErrWithCause instead of defining a new error?

Reply:
Sometimes there are many error When opening shard, I cant find a correct source error for ErrWithCause.

Reply:
How about ErrNoCause?

Reply:
ok